common-lib/test/cmr/common/test/api/errors.clj

```diff
@@ -0,0 +1,12 @@
+(ns cmr.common.test.api.web-server
+  "This tests capabilities of the web server component."
+  (:require
+   [clj-http.client :as h]
+   [clojure.java.io :as io]
+   [clojure.string :as str]
+   [clojure.test :refer :all]
+   [cmr.common.api.errors :as errors]))
+
+(deftest test-error-masked
+  (let [error-message "Token 123 does not exist"]
+  (is (= "Token does not exist" (errors/mask-token-error error-message)))))
```

Comment:
Nit: indent so the `is` is within the `let`:
```
(let [error-message "Token 123 does not exist"]
  (is (= "Token does not exist" (errors/mask-token-error error-message)))))
```

Reply:
good call - I should have noticed that! https://github.com/nasa/Common-Metadata-Repository/pull/738/commits/2df88609ba6592a005da00251391fcb3e2364ff6